source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,26 @@
+=====================
+Update Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+You can update multiple documents using the ``updateMany()`` method. 
+``updateMany()`` accepts a filter object, and updates the first document
+that matches the filter using a provided update document. The update
+document requires an :manual:`Update Operator
+</reference/operator/update>` to modify a field in a document.
+
+Create an `Object
```

Comment:
"Create an Object" might be confusing to readers since it doesn't actually tell users how these options are passed into the method. It might be easier to understand if we say something more like "pass the optional options parameter" or "Pass an object as the optional third parameter to ``updateMany()`` to specify additional options".

Reply:
Originally it was 'create an additional options object' as part of another PR, but someone suggested this current sentence. Notice, this is the same sentence for this as many of the other usage-examples. I'd like to keep it in the current format for this PR (atleast for the sake of this PR), and then once we do a once over - consider changing the sentence for all of the usage-examples that have it. 

I also, think it's not particularly confusing with the example snippet (especially once we get the 2 column layout)